src/pages/database-main/DatabaseContent.jsx

```diff
@@ -20,13 +20,31 @@ const sideMenu = sideMenuItems.map((key) => ({
   label: `Sider ${key}`,
 }));
 
+const arrayTemp = ["A", "B", "C"];
 const DatabaseContent = () => {
   const [selectedKey, setSelectedKey] = useState(sideMenuItems[0]);
+  const [searchValue, setSearchValue] = useState("");
+  const filteredResults = arrayTemp.filter((value) => 
+    value.toLowerCase().includes(searchValue.toLowerCase())
+  );
 
   return (
     <>
       <div>
-        Full lenegth Search Bar here (fit the whole width, TODO: Curwen)
+        <Input 
```

Comment:
don't just implement the searchbar in this component, 

create a dedicated folder and file for the search bar under `components` directory.

create a file called `searchbar`, and create a new file name `SearchBar.jsx` and do the implementation there.

we want to make sure the code is modular.